### PR TITLE
Fix contributing link

### PR DESCRIPTION
**Pull request by NoriSte**
**Summary**
The yarn 2 website does not redirect the `org/contributing` URL.

### CODE REVIEW

**Comment by olingern** [comment]
Also, I should note that the cookies ( I suppose that's what's doing the legacy redirect) seem kind of sticky and I can't directly navigate to the v2 docs I've I've chosen to go to the legacy docs in the past.

ping @Daniel15 @arcanis

**Comment by PierreBerger** [comment]
There is also the Contributing link of the [README](https://github.com/yarnpkg/yarn/blob/master/README.md#contributing-to-yarn) to replace.

**Comment by NoriSte** [comment]
Ok (for the README) but what about the @oligot note?
More: is the "legacy" third-level domain going to be renamed "classic"?

**Comment by Daniel15** [comment]
@NoriSte The domain has already been changed... I might set up a 301 redirect from legacy. to classic. at some point. Currently both domains still work.

**Comment by NoriSte** [comment]
The README.md' link goes to the CONTRIBUTING.md file where I've added both the v1 and the v2 links 😊

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1 +1,3 @@
-Please see https://yarnpkg.com/org/contributing/
\ No newline at end of file
+Contributing to Yarn V1: please see https://classic.yarnpkg.com/en/docs/getting-started
+
+Contributing to Yarn V2: please see https://yarnpkg.com/en/org/contributing
```
Comment (Tom-Bonnike) [inline CONTRIBUTING.md]: FYI,  that link somehow redirects to the classic website (V1 contributing link). I don’t think that’s expected? Shouldn’t it be https://yarnpkg.com/advanced/contributing?

**Comment by Ende93** [comment]
@NoriSte  Seem like the old request change blocks the merge, can you resolve it?

**Comment by NoriSte** [comment]
Hi @Ende93 It's not clear to me why the `continuous-integration/appveyor/pr` build is failing, could you help me, please?
I don't understand how the 
```
FAIL __tests__\commands\add.js (83.861s)
  ● add with --ignore-workspace-root-check shouldn't fail on the workspace root
    Error: expect(received).toBe(expected) // Object.is equality
```
test could be related to my changes...

**Comment by Ende93** [comment]
I saw [the lastest commit(Fixes empty yarnrc files)](https://github.com/yarnpkg/yarn/commit/a75dbe36ca7f6d881d291a7231676d00d523bb39) also failed the same test, so it's ok.

**Comment by NoriSte** [comment]
Ok, let me know if you need me adding more changes 😊

**Comment by Ende93** [comment]
Please resolve the request.
image

**Comment by Andrew836-dev** [comment]
This seems like the preferred solution to these open issues
#8426 #8350 #8335 #7950
and these open pull requests
#8427 #8365 #8356 #8336 #8213 #8189 #8159 #7952 #7494

@arcanis @olingern

**Comment by NoriSte** [comment]
I left following the PR because it's not clear to me if I have to do something more or not... 🤔

**Review by alanrps — APPROVED** [review]

**Comment by merceyz** [comment]
Fixed in https://github.com/yarnpkg/yarn/commit/0a121f66ffcb4fc48ed6ea9e5ad928bd7cdc9b5f

**Pull request closed without merging**